core/src/main/scala/sttp/openai/requests/completions/chat/ChatRequestBody.scala

```diff
@@ -45,7 +63,8 @@ object ChatRequestBody {
       presencePenalty: Option[Double] = None,
       frequencyPenalty: Option[Double] = None,
       logitBias: Option[Map[String, Float]] = None,
-      user: Option[String] = None
+      user: Option[String] = None,
+      stream: Option[Boolean] = None
```

Comment:
You're right, it might be confusing. I completely removed the stream parameter from `ChatBody` and created `ChatBody.withStreaming` method which directly adds `"stream": true` field to json when the createChatCompletion with streams is invoked.